### ISSUE TITLE
resSeq attribute not set by mdtraj.Topology.from_openmm()

### ISSUE THREAD

**Issue by JBGreisman** [mdtraj/mdtraj#1422]
Hi -- 

For system prep I have been using Modeller in OpenMM, and I have been using `mdtraj.Topology.from_openmm()` occasionally to use atom selections. However, it seems that the `Residue.resSeq` attribute is not set when building the Topology with `mdtraj.Topology.from_openmm()`. 

Is it possible to change this line in `from_openmm()` in mdtraj/core/topology.py:
```
...
for residue in chain.residues():
    r = out.add_residue(residue.name, c)
...
```
to the following
```
...
for residue in chain.residues():
    r = out.add_residue(residue.name, c, residue.id)
...
```
?

The `Residue.resSeq` attribute seems to be set when I load a pdb directly in mdtraj using `load_pdb()`, so it doesn't seem like this change is likely to have side effects. Please let me know if you think there's anything I'm missing here.

**Comment by rmcgibbo** [comment]
If you proposed a pull request with that change, I'd merge it.

**Comment by JBGreisman** [comment]
Closing issue: resolved in pull request: #1424
